datasetinsights/stats/visualization/keypoints_plot.py

```diff
@@ -0,0 +1,174 @@
+""" Helper keypoints library to plot keypoint joints and skeletons  with a
+simple Python API.
+"""
+
+
+def get_color_from_color_node(color):
```

Comment:
This should be a protected method start with underscore like `_get_color_from_color_node` to let users know this is a helper function instead of officially supported API.

Reply:
Ok, fixed